wiki/Glossary/Break/en.md

```diff
@@ -0,0 +1,16 @@
+---
+tags:
+  - pause
+  - Break Time
+---
+
+# Break
+
+*Break* or **Break Time**, is a term used when a section of a [beatmap](/wiki/beatmaps) has no [hit objects](/wiki/Hit_Objects). During this time, the drain in the health bar is temporarily halted. Breaks usually allow the player to rest their arm for a short while and reposition their input device.
```

Comment:
"break" and "break time" aren't alternatives of each other, it's just different ways to express this idea

> **Breaks** are sections of a beatmap [...]